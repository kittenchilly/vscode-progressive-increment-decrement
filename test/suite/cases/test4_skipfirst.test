##skipFirstNumber
0 10 23