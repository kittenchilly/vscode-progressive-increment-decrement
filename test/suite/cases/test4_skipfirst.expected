##skipFirstNumber
0 01 02